spec/features/accordion_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+describe "Accordion", js: true, type: :view do
+  scenario "There is a page with the accordion rendered" do
+    given_i_visit_a_page_with_the_accordion_component
+    then_the_accordion_loads
+    then_the_accordion__javascript_loads
+    when_i_click_the_first_accordion_section
+    then_the_accordion_opens
+    when_i_click_the_first_accordion_section
+    then_the_accordion_has_data_attributes
+  end
+
+  def given_i_visit_a_page_with_the_accordion_component
+    visit "/accordionexample"
+  end
+
+  def then_the_accordion_loads
+    expect(page).to have_css(".gem-c-accordion", visible: true)
+  end
+
+  def then_the_accordion__javascript_loads
+    expect(page).to have_css(".gem-c-accordion--active", visible: true)
+  end
+
+  def when_i_click_the_first_accordion_section
+    page.first(".govuk-accordion__section-button").click
+  end
+
+  def then_the_accordion_opens
+    puts page.body
+    expect(page).to have_selector(".govuk-accordion__section.govuk-accordion__section--expanded", visible: true)
```

Comment:
Could we also check for the content of the open section? That way we know that the CSS is doing the correct thing.

For example:
```ruby
expect(page).to have_content("This is the content for Writing well for the web.")
```